docs/pages/api.rst

```diff
@@ -454,16 +471,16 @@ Signature: ``Collection::current(int $index = 0);``
 cycle
 ~~~~~
 
-Cycle around a collection of items.
+Cycle indefinitely around a collection of items.
 
 Interface: `Cycleable`_
 
-Signature: ``Collection::cycle(int $length = 0);``
```

Comment:
I'm not sure if this was for an older version, but the current `cycle` method doesn't take a parameter and cycles indefinitely from what I can see

Reply:
Seems like this was from an old implementation. Maybe we should keep the signature, but just updating it:

` Signature: ``Collection::cycle();`` `